src/WooCommerce.php

```diff
@@ -16,6 +16,11 @@ class WooCommerce {
    * @implements admin_init
    */
   public static function init() {
+    // Add WooCommerce v3.x Gallery Lightbox support
+    // @see https://woocommerce.wordpress.com/2017/02/28/adding-support-for-woocommerce-2-7s-new-gallery-feature-to-your-theme/
+    // @see https://github.com/woocommerce/woocommerce/wiki/Enabling-product-gallery-features-(zoom,-swipe,-lightbox)-in-3.0.0
+    add_theme_support('wc-product-gallery-lightbox');
```

Comment:
According to https://woocommerce.wordpress.com/2017/02/28/adding-support-for-woocommerce-2-7s-new-gallery-feature-to-your-theme/ declaring this support will actually re-instantiate the default/built-in lightbox of WooCommerce.

But isn't the whole point of the gallerya plugin to replace the built-in lightbox?